pandas/io/html.py

```diff
@@ -352,13 +354,13 @@ def _parse_tfoot_tr(self, table):
         """
         raise AbstractMethodError(self)
 
-    def _parse_tables(self, doc, match, attrs):
+    def _parse_tables(self, dom_doc, match, attrs):
```

Comment:
this seems irrelevant (and I don't know if it affects anyone who might be subclassing) - can we revert this please?

Reply:
Thanks for the suggestion. I made this change because I thought there was some conflict between the `doc` parameter in the `_parse_tables` function and the `@doc` decorator being applied, but now that you've suggested to just go in and change each instance anyway, this change will certainly be reverted along with the others so no worries there.